patches/server/0981-Add-custom-destroyerIdentity-to-sendBlockDamage.patch

```diff
@@ -0,0 +1,30 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: TheFruxz <cedricspitzer@outlook.de>
+Date: Sat, 26 Mar 2022 11:52:17 +0100
+Subject: [PATCH] Add custom destroyerIdentity to sendBlockDamage
+
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java b/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
+index 6feeadad9aecb7d63e24d5daae115a93e39aeb3d..3027900889c43369aee054fa54d8f1aa77e28505 100644
+--- a/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
++++ b/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
+@@ -917,13 +917,18 @@ public class CraftPlayer extends CraftHumanEntity implements Player {
+ 
+     @Override
+     public void sendBlockDamage(Location loc, float progress) {
++        sendBlockDamage(loc, progress, this.getHandle().getId()); // Paper start - customBlockDamage identity
++    }
++
++    @Override
++    public void sendBlockDamage(Location loc, float progress, int destroyerIdentity) { // Paper end - customBlockDamage identity
```

Comment:
Throw those paper start and end comments above and below those lines respectively. Basically, you want the `// Paper start` and `// Paper end` comments to perfectly wrap the diff when you look at this patch, so the first line of the diff should have the start comment only, and the last has the end comment only.

Reply:
now seperate lines